README.md

```diff
@@ -29,4 +29,15 @@ KEY TAKEAWAYS:
 Limit Wildcard Imports  
 Force braces (No one line while, if, etc)  
 Indentations  
-Formatting for imports  
\ No newline at end of file
+Formatting for imports  
+
+
+## Guide to intial building with Maven
```

Comment:
What is with the backslashes ``\``?  
If ye want to go to a new line then its two spaces and then enter

Reply:
mfw when iv been doing ``\`` all this time